widgets/src/main/java/org/pentaho/gwt/widgets/client/tabs/PentahoTab.java

```diff
@@ -185,4 +188,23 @@ public boolean isSolutionBrowserShowing() {
   public void setSolutionBrowserShowing( boolean solutionBrowserShowing ) {
     this.solutionBrowserShowing = solutionBrowserShowing;
   }
+  @Override
+  public int getTabIndex() {
+    return focusImpl.getTabIndex(this.getElement());
+  }
+  @Override
+  public void setAccessKey(char key) {
+    this.getElement().setPropertyString("accessKey", "" + key);
+  }
+  @Override
+  public void setFocus(boolean focus ) {
+    if (focus) {
+      focusImpl.focus(this.getElement());
+    }
+  }
+  @Override
+  public void setTabIndex(int index) {
+    focusImpl.setTabIndex(this.getElement(), index);
+  }
+
```

Comment:
Need a new line between methods. And no need for new line after the last one.
Also, spacing is off.